utils/cLog.cpp

```diff
@@ -165,12 +165,12 @@ void cLogger::log(eLogLevel level, eLogComponent component, const std::string& e
 
     logline += event;
 
-    fprintf(m_file, "%s\n", logline.c_str()); // print the text into the file
-    fflush(m_file);
+    logCommentLine(logline);
 }
 
 void cLogger::logCommentLine(const std::string& txt) {
-    fprintf(m_file, "\\\\%s\n", txt.c_str()); // print the text into the file
+    m_file << txt << std::endl;
```

Comment:
```suggestion
    m_file << "\\" << txt << std::endl;
```
This is some feature that Stefan likes. There are lines like this in the logfile:
```
\\-------
\\Allegro
\\-------
```

Reply:
Oups. i fix as previously

Reply:
haha true. I like to have some 'headers' to tell when certain parts of logging start... so yeah please keep that ;-)